<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>807</width>
    <height>451</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: qlineargradient(spread:pad, x1:1, y1:1, x2:0, y2:0, stop:0 rgba(81, 0, 135, 255), stop:0.427447 rgba(41, 61, 132, 235), stop:1 rgba(155, 79, 165, 255));</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="styleSheet">
    <string notr="true">color: white;</string>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QFrame" name="fr_add">
        <property name="styleSheet">
         <string notr="true">background-color: rgba(255,255,255,30);
border: 1px solid rgba(255,255,255,40);
border-top-right-radius: 7px; 
border-top-left-radius: 7px; </string>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <widget class="QLineEdit" name="le_user_pass">
           <property name="styleSheet">
            <string notr="true">font-size: 16pt;
color: white;
padding-left: 10px;
border-radius:7px;</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="btn_add">
           <property name="styleSheet">
            <string notr="true">QPushButton{
font-size: 12pt;
	color: rgb(255, 255, 255);
     background-color:rgba(255,255,255,30);
     border: 1px solid rgba(255,255,255,40);
     border-radius:7px;
width: 100;
height: 30;
}
QPushButton:hover{
background-color:rgba(255,255,255,30);
}
QPushButton:pressed{
background-color:rgba(255,255,255,70);
}</string>
           </property>
           <property name="text">
            <string>Add</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="btn_login">
           <property name="styleSheet">
            <string notr="true">QPushButton{
font-size:12pt;
	color: rgb(255, 255, 255);
     background-color:rgba(255,255,255,30);
     border: 1px solid rgba(255,255,255,40);
     border-radius:7px;
width: 100;
height: 30;
}
QPushButton:hover{
background-color:rgba(255,255,255,30);
}
QPushButton:pressed{
background-color:rgba(255,255,255,70);
}</string>
           </property>
           <property name="text">
            <string>Login</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QTableView" name="tableView">
        <property name="styleSheet">
         <string notr="true">QTableView {
background-color: rgba(255, 255, 255, 30); 
border: 1px solid rgba(255,255,255,40);
border-bottom-right-radius: 7px; 
border-bottom-left-radius: 7px; 
color: white;
}

QHeaderView::section {
background-color: rgb(53, 53, 53);
color: white;
border: none;
height: 50px;
width: 230;
font-size: 16pt;
}

QTableView::item {
    border-style: none;
    border-bottom: 1px solid rgba(255,255,255,50);
    padding-left: auto;
    padding-right: auto;
}

QTableView::item:selected{
	border: none;
	color: rgb(255, 255, 255);
    background-color: rgba(255, 255, 255, 50);
}
</string>
        </property>
        <property name="showGrid">
         <bool>false</bool>
        </property>
        <property name="gridStyle">
         <enum>Qt::NoPen</enum>
        </property>
        <property name="sortingEnabled">
         <bool>true</bool>
        </property>
        <attribute name="horizontalHeaderCascadingSectionResizes">
         <bool>true</bool>
        </attribute>
        <attribute name="horizontalHeaderDefaultSectionSize">
         <number>150</number>
        </attribute>
        <attribute name="horizontalHeaderShowSortIndicator" stdset="0">
         <bool>true</bool>
        </attribute>
        <attribute name="horizontalHeaderStretchLastSection">
         <bool>true</bool>
        </attribute>
        <attribute name="verticalHeaderVisible">
         <bool>false</bool>
        </attribute>
        <attribute name="verticalHeaderHighlightSections">
         <bool>false</bool>
        </attribute>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources>
  <include location="resource.qrc"/>
 </resources>
 <connections/>
</ui>
